src/navigations/HomeBottomTabs.js

```diff
@@ -186,6 +190,11 @@ function HomeBottomTabs({navigation}) {
       handlePushNotif(remoteMessage);
     });
 
+    messaging().setBackgroundMessageHandler(async (remoteMessage) => {
+      console.log('Message handled in the background!', remoteMessage);
+      handlePushNotif(remoteMessage);
+    });
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A background message handler has been added for Firebase messaging. This is a good practice for handling push notifications when the app is in the background. However, ensure that the `handlePushNotif` function can handle messages correctly in both foreground and background states.



<!-- This is an auto-generated comment by OSS CodeRabbit -->